src/lib/viewers/media/MediaBaseViewer.js

```diff
@@ -464,23 +474,48 @@ class MediaBaseViewer extends BaseViewer {
     /**
      * Loads the controls
      *
-     * @private
+     * @protected
      * @return {void}
      */
     loadUI() {
         this.mediaControls = new MediaControls(this.mediaContainerEl, this.mediaEl, this.cache);
 
-        // Add event listeners for the media controls
         this.addEventListenersForMediaControls();
+        this.addEventListenersForMediaElement();
+    }
+
+    /**
+     * Loads the React controls
+     *
+     * @protected
+     * @return {void}
+     */
+    loadUIReact() {
+        if (!this.cache.has(MEDIA_AUTOPLAY_CACHE_KEY)) {
+            this.cache.set(MEDIA_AUTOPLAY_CACHE_KEY, 'Disabled');
+        }
 
-        // Add event listeners for the media element
+        if (!this.cache.has(MEDIA_SPEED_CACHE_KEY)) {
+            this.cache.set(MEDIA_SPEED_CACHE_KEY, '1.0');
+        }
+
+        this.controls = new MediaControlsRoot({ containerEl: this.containerEl });
```

Comment:
This will likely end up renamed/moved to the MP3Viewer once the video controls are introduced, since the latter has auto-hide/show behavior similar to the document viewers.